<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1074</width>
    <height>814</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1040</width>
      <height>799</height>
     </rect>
    </property>
    <property name="title">
     <string>Particula</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_20">
     <item row="0" column="2">
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>Tabla</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="1">
          <widget class="QPushButton" name="buscar_pushButton">
           <property name="text">
            <string>Buscar</string>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QPushButton" name="sortDist_pushButton">
           <property name="text">
            <string>Ordenar Distancia</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="mostrar_tabla_pushButton">
           <property name="text">
            <string>Mostrar</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="6">
          <widget class="QTableWidget" name="tabla"/>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="id_lineEdit">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>ID de la particula</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QPushButton" name="sortVel_pushButton">
           <property name="text">
            <string>Ordenar Velocidad</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QPushButton" name="sortId_pushButton">
           <property name="text">
            <string>Ordenar Id</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_7">
        <attribute name="title">
         <string>Gr√°ficar</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="1" column="0">
          <widget class="QPushButton" name="dibujar">
           <property name="text">
            <string>Dibujar</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="limpiar">
           <property name="text">
            <string>Limpiar</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QGraphicsView" name="graphicsView"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QGroupBox" name="groupBox_10">
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_17"/>
      </widget>
     </item>
     <item row="0" column="0" rowspan="3" colspan="2">
      <widget class="QGroupBox" name="groupBox_11">
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_19">
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>Ordenar</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_14">
           <item row="0" column="1">
            <widget class="QSpinBox" name="origen_x_spinBox">
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Y</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QSpinBox" name="origen_y_spinBox">
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>X</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QGroupBox" name="groupBox_8">
          <property name="title">
           <string>Destino</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_15">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>X</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="destino_x_spinBox">
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Y</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QSpinBox" name="destino_y_spinBox">
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QGroupBox" name="groupBox_9">
          <property name="title">
           <string>Color</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>R</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="red_spinBox">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>G</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QSpinBox" name="green_spinBox">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>B</string>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <widget class="QSpinBox" name="blue_spinBox">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QGroupBox" name="groupBox_12">
          <property name="title">
           <string/>
          </property>
          <layout class="QGridLayout" name="gridLayout_18">
           <item row="0" column="1">
            <widget class="QSpinBox" name="velocidad_spinBox">
             <property name="maximum">
              <number>10000</number>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_37">
             <property name="text">
              <string>Velocidad</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QGroupBox" name="groupBox_13">
          <property name="title">
           <string>Salida</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0" colspan="2">
            <widget class="QPlainTextEdit" name="salida_plainTextEdit"/>
           </item>
           <item row="1" column="0">
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Ordenar</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="2" column="0">
               <widget class="QPushButton" name="ordenar_velocidad_pushButton">
                <property name="text">
                 <string>Velocidad</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QPushButton" name="ordenar_distancia_pushButton">
                <property name="text">
                 <string>Distancia</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QPushButton" name="ordenar_id_pushButton">
                <property name="text">
                 <string>Id</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QGroupBox" name="groupBox_14">
             <property name="title">
              <string>Agregar</string>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="3" column="0" colspan="2">
               <widget class="QPushButton" name="agregar_final_pushButton">
                <property name="text">
                 <string>Agregar Final</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QPushButton" name="agregar_inicio_pushButton">
                <property name="text">
                 <string>Agregar Inicio</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QPushButton" name="mostrar_pushButton">
             <property name="text">
              <string>Mostrar</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string/>
          </property>
          <layout class="QGridLayout" name="gridLayout_13">
           <item row="0" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>ID</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="id_spinBox">
             <property name="maximum">
              <number>100000</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1074</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
   </widget>
   <widget class="QMenu" name="menuVer">
    <property name="title">
     <string>Ver</string>
    </property>
    <addaction name="actionPuntos"/>
    <addaction name="actionMostrar_Grafo"/>
   </widget>
   <widget class="QMenu" name="menuAlgoritmos">
    <property name="title">
     <string>Algoritmos</string>
    </property>
    <addaction name="actionPuntos_mas_cercanos"/>
    <addaction name="actionRecorrido_de_Grafo"/>
    <addaction name="actionPrim"/>
    <addaction name="actionKruskal"/>
    <addaction name="actionDijkstra"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuVer"/>
   <addaction name="menuAlgoritmos"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir Archivo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar Archivo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionPuntos">
   <property name="text">
    <string>Puntos</string>
   </property>
  </action>
  <action name="actionPuntos_mas_cercanos">
   <property name="text">
    <string>Puntos mas cercanos</string>
   </property>
  </action>
  <action name="actionMostrar_Grafo">
   <property name="text">
    <string>Mostrar Grafo</string>
   </property>
  </action>
  <action name="actionRecorrido_de_Grafo">
   <property name="text">
    <string>Recorrido de Grafo</string>
   </property>
  </action>
  <action name="actionPrim">
   <property name="text">
    <string>Prim</string>
   </property>
  </action>
  <action name="actionKruskal">
   <property name="text">
    <string>Kruskal</string>
   </property>
  </action>
  <action name="actionDijkstra">
   <property name="text">
    <string>Dijkstra</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
